prod-sample/sample-modules/cultured-frankie/0.0.0/.gitignore

```diff
@@ -1,6 +1,7 @@
 # node/npm
 node_modules
 *.log
+package-lock.json
```

Comment:
We really shouldn't have individual `.gitignore`s for each of these, can you consolidate?

Reply:
done